aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.h

```diff
@@ -57,7 +61,7 @@ void ldl_solve_cusolver(
 void lu_factor_batched_cublas(const Tensor& A, const Tensor& pivots, const Tensor& infos, bool get_pivots);
 void lu_solve_batched_cublas(const Tensor& LU, const Tensor& pivots, const Tensor& B, TransposeType transpose);
 
-#ifdef USE_CUSOLVER
+#if defined(USE_CUSOLVER) || defined(USE_HIPSOLVER)
```

Comment:
I believe I've left this comment already, but why two defines is needed here? Why not `#ifdef USE_GPU_SOLVER` which in the common header file is define for both CUDA and ROCm platforms?

Reply:
woops x 2. Meant to address this last time.